src/appsettings/settings.py

```diff
@@ -607,6 +624,20 @@ def __init__(
             validators=validators,
         )
 
+    def decode_environ(self, value):
+        """
+        Return a boolean decoded value.
+
+        Return True if the value is "True" or "1", False otherwise.
+
+        Args:
+            value (str):
+
+        Returns:
+            bool:
+        """
+        return value == "True" or value == "1"
```

Comment:
`true` (lowercase) is also a common value among env vars. Other commonly used values include `yes` or `on`.

There should also be some kind of control for the negative values and `ImproperlyConfigured` exception should be thrown if any other value than those listed is provided. In other words, it's not a good idea to just suppose that every other values is `False`.

Reply:
Fixed